Assets/UGSSamples/FriendsSample/Scripts/Common/Managers/RelationshipsManager.cs

```diff
@@ -17,19 +17,20 @@ public class RelationshipsManager : MonoBehaviour
         GameObject m_RelationshipsViewGameObject; //This gameObject reference is only needed to get the IRelationshipUIController component from it.
         IRelationshipsView m_RelationshipsView;
 
-        List<FriendsEntryData> m_FriendsEntryDatas = new List<FriendsEntryData>();
-        List<PlayerProfile> m_RequestsEntryDatas = new List<PlayerProfile>();
-        List<PlayerProfile> m_BlockEntryDatas = new List<PlayerProfile>();
+        List<FriendsEntryData> m_FriendsEntryDatas = new ();
+        List<PlayerProfile> m_RequestsEntryDatas = new ();
+        List<PlayerProfile> m_BlockEntryDatas = new ();
 
-        string m_LoggedPlayerName;
         ILocalPlayerView m_LocalPlayerView;
         IAddFriendView m_AddFriendView;
         IFriendsListView m_FriendsListView;
         IRequestListView m_RequestListView;
         IBlockedListView m_BlockListView;
-        ISocialProfileService m_SocialProfileService;
+        
+        IPlayerProfileService m_SamplePlayerProfileService;
         IManagedRelationshipService m_ManagedRelationshipService;
 
+        string LoggedPlayerName => m_SamplePlayerProfileService.GetName(LoggedPlayerId);
```

Comment:
should we not have a PlayerProfile for the player that we set on login that holds ID and name?
PlayerProfile m_LocalPlayer; etc